Cannot start Docker: ModuleNotFoundError: No module named 'llama_cpp'

### Bug description

I am using docker desktop for MacOS and only receiving that error when i try to start the docker.

### Steps to reproduce

1. Try to install serge
2.get the error

### Environment Information

OS: MacOS
CPU: M1

### Screenshots

_No response_

### Relevant log output

_No response_

### Confirmations

- [X] I'm running the latest version of the main branch.
- [X] I checked existing issues to see if this has already been described.

same issue same error too sharing if it is useful

```
2023-06-21 18:07:19       FAILED: /tmp/pip-install-7db_7iw8/llama-cpp-python_b6bf82ea03c7458892951ca7ada18e83/vendor/llama.cpp/libllama.so
2023-06-21 18:07:19       cd /tmp/pip-install-7db_7iw8/llama-cpp-python_b6bf82ea03c7458892951ca7ada18e83/vendor/llama.cpp && make libllama.so
2023-06-21 18:07:19       I llama.cpp build info:
2023-06-21 18:07:19       I UNAME_S:  Linux
2023-06-21 18:07:19       I UNAME_P:  unknown
2023-06-21 18:07:19       I UNAME_M:  aarch64
2023-06-21 18:07:19       I CFLAGS:   -I.              -O3 -std=c11   -fPIC -DNDEBUG -Wall -Wextra -Wpedantic -Wcast-qual -Wdouble-promotion -Wshadow -Wstrict-prototypes -Wpointer-arith -pthread -DGGML_USE_K_QUANTS -mcpu=native
2023-06-21 18:07:19       I CXXFLAGS: -I. -I./examples -O3 -std=c++11 -fPIC -DNDEBUG -Wall -Wextra -Wpedantic -Wcast-qual -Wno-unused-function -Wno-multichar -pthread -mcpu=native
2023-06-21 18:07:19       I LDFLAGS:
2023-06-21 18:07:19       I CC:       cc (Debian 12.2.0-14) 12.2.0
2023-06-21 18:07:19       I CXX:      g++ (Debian 12.2.0-14) 12.2.0
2023-06-21 18:07:19       
2023-06-21 18:07:19       g++ -I. -I./examples -O3 -std=c++11 -fPIC -DNDEBUG -Wall -Wextra -Wpedantic -Wcast-qual -Wno-unused-function -Wno-multichar -pthread -mcpu=native -c llama.cpp -o llama.o
2023-06-21 18:07:19       cc  -I.              -O3 -std=c11   -fPIC -DNDEBUG -Wall -Wextra -Wpedantic -Wcast-qual -Wdouble-promotion -Wshadow -Wstrict-prototypes -Wpointer-arith -pthread -DGGML_USE_K_QUANTS -mcpu=native   -c ggml.c -o ggml.o
2023-06-21 18:07:19       In file included from ggml.c:180:
2023-06-21 18:07:19       /usr/lib/gcc/aarch64-linux-gnu/12/include/arm_neon.h: In function ‘ggml_vec_dot_f16’:
2023-06-21 18:07:19       /usr/lib/gcc/aarch64-linux-gnu/12/include/arm_neon.h:29182:1: error: inlining failed in call to ‘always_inline’ ‘vfmaq_f16’: target specific option mismatch
2023-06-21 18:07:19       29182 | vfmaq_f16 (float16x8_t __a, float16x8_t __b, float16x8_t __c)
2023-06-21 18:07:19             | ^~~~~~~~~

.........

2023-06-21 18:07:19       make: *** [Makefile:249: ggml.o] Error 1
2023-06-21 18:07:19       ninja: build stopped: subcommand failed.
2023-06-21 18:07:19       Traceback (most recent call last):
2023-06-21 18:07:19         File "/tmp/pip-build-env-53a4eqb0/overlay/lib/python3.11/site-packages/skbuild/setuptools_wrap.py", line 674, in setup
2023-06-21 18:07:19           cmkr.make(make_args, install_target=cmake_install_target, env=env)
2023-06-21 18:07:19         File "/tmp/pip-build-env-53a4eqb0/overlay/lib/python3.11/site-packages/skbuild/cmaker.py", line 697, in make
2023-06-21 18:07:19           self.make_impl(clargs=clargs, config=config, source_dir=source_dir, install_target=install_target, env=env)
2023-06-21 18:07:19         File "/tmp/pip-build-env-53a4eqb0/overlay/lib/python3.11/site-packages/skbuild/cmaker.py", line 742, in make_impl
2023-06-21 18:07:19           raise SKBuildError(msg)
2023-06-21 18:07:19       
2023-06-21 18:07:19       An error occurred while building with CMake.
2023-06-21 18:07:19         Command:
2023-06-21 18:07:19           /tmp/pip-build-env-53a4eqb0/overlay/lib/python3.11/site-packages/cmake/data/bin/cmake --build . --target install --config Release --
2023-06-21 18:07:19         Install target:
2023-06-21 18:07:19           install
2023-06-21 18:07:19         Source directory:
2023-06-21 18:07:19           /tmp/pip-install-7db_7iw8/llama-cpp-python_b6bf82ea03c7458892951ca7ada18e83
2023-06-21 18:07:19         Working directory:
2023-06-21 18:07:19           /tmp/pip-install-7db_7iw8/llama-cpp-python_b6bf82ea03c7458892951ca7ada18e83/_skbuild/linux-aarch64-3.11/cmake-build
2023-06-21 18:07:19       Please check the install target is valid and see CMake's output for more information.
2023-06-21 18:07:19       
2023-06-21 18:07:19       [end of output]
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,12 +1,12 @@
 # ---------------------------------------
 # Base image for node
-FROM node:19-slim as node_base
+FROM node:19-bullseye-slim as node_base
 
 WORKDIR /usr/src/app
 
 # ---------------------------------------
 # Base image for runtime
-FROM python:3.11-slim as base
+FROM python:3.11-slim-bullseye as base
 
 ENV TZ=Etc/UTC
 WORKDIR /usr/src/app

--- a/api/src/serge/data/models.json
+++ b/api/src/serge/data/models.json
@@ -72,14 +72,9 @@
                 "repo": "TheBloke/chronos-hermes-13B-GGML",
                 "files": [
                     {
-                        "name": "q2_K",
-                        "filename": "chronos-hermes-13b.ggmlv3.q2_K.bin",
-                        "disk_space": 55100000000.0
-                    },
-                    {
-                        "name": "q3_K_L",
-                        "filename": "chronos-hermes-13b.ggmlv3.q3_K_L.bin",
-                        "disk_space": 69300000000.0
+                        "name": "q4_0",
+                        "filename": "chronos-hermes-13b.ggmlv3.q4_0.bin",
+                        "disk_space": 73200000000.0
                     },
                     {
                         "name": "q4_1",
@@ -87,24 +82,14 @@
                         "disk_space": 81400000000.0
                     },
                     {
-                        "name": "q4_K_M",
-                        "filename": "chronos-hermes-13b.ggmlv3.q4_K_M.bin",
-                        "disk_space": 78700000000.0
+                        "name": "q5_0",
+                        "filename": "chronos-hermes-13b.ggmlv3.q5_0.bin",
+                        "disk_space": 89500000000.0
                     },
                     {
                         "name": "q5_1",
                         "filename": "chronos-hermes-13b.ggmlv3.q5_1.bin",
                         "disk_space": 97600000000.0
-                    },
-                    {
-                        "name": "q5_K_M",
-                        "filename": "chronos-hermes-13b.ggmlv3.q5_K_M.bin",
-                        "disk_space": 92300000000.0
-                    },
-                    {
-                        "name": "q6_K",
-                        "filename": "chronos-hermes-13b.ggmlv3.q6_K.bin",
-                        "disk_space": 10700000000.0
                     },
                     {
                       "name": "q8_0",

